data/lib/core/player.lua

```diff
@@ -383,3 +383,7 @@ function Player.setSpecialContainersAvailable(self, available)
 	msg:delete()
 	return true
 end
+
+function Player.addBankMoney(self, amount)
```

Comment:
Keep the pattern and name it `addBankBalance`
```suggestion
function Player.addBankBalance(self, amount)
```